<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtractSeeds</class>
 <widget class="qSlicerWidget" name="ExtractSeeds">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExtractSkeleton</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="ctkCollapsibleGroupBox" name="ExtractSeedsCollapsibleGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>A) Extract seeds</string>
     </property>
     <property name="collapsedHeight">
      <number>5</number>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="ExtractSeedsMaskLabel">
        <property name="text">
         <string>Organ mask</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="ExtractSeedsInputNodeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the fixed image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Fixed</string>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="ExtractSeedsInputLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Enhance image</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="ExtractSeedsMaskNodeComboBox">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="removeEnabled">
         <bool>true</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="8" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="ExtractSeedsGoToModulePushButton">
          <property name="toolTip">
           <string>Open the &quot;BrainsFit&quot; module</string>
          </property>
          <property name="text">
           <string>Go to Extract Skeleton module</string>
          </property>
          <property name="workflow" stdset="0">
           <stringlist>
            <string>2</string>
           </stringlist>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ExtractSeedsApplyPushButton">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Extract</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ExtractSeedsSavePushButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save result </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="ExtractSeedsOutputLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Extracted image</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="ExtractSeedsObjectIDLabel">
        <property name="text">
         <string>Organ label</string>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="qMRMLLabelComboBox" name="ExtractSeedsObjectIDLabelComboBox">
        <property name="workflow" stdset="0">
         <stringlist>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="qMRMLNodeComboBox" name="ExtractSeedsOutputNodeComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Select the fixed image</string>
        </property>
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="baseName">
         <string>Fixed</string>
        </property>
        <property name="addEnabled">
         <bool>true</bool>
        </property>
        <property name="editEnabled">
         <bool>true</bool>
        </property>
        <property name="workflow" stdset="0">
         <stringlist>
          <string>1</string>
          <string>2</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLLabelComboBox</class>
   <extends>qMRMLWidget</extends>
   <header>qMRMLLabelComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>ctkCollapsibleGroupBox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ExtractSeeds</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ExtractSeedsInputNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExtractSeeds</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ExtractSeedsMaskNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ExtractSeeds</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>ExtractSeedsOutputNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
